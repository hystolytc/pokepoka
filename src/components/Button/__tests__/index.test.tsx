import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'
import { Button, ButtonStickyBottom } from '../index'

const describeButtonComponent = '<Button /> test case'
describe(describeButtonComponent, () => {
  const { container } = render(
    <Button
      type='stretch'
      variant='primary'
      size='sm'
      onClick={() => {}}
    >
      Button test
    </Button>
  )

  test(`Render ${describeButtonComponent} correctly`, () => {
    expect(container).toMatchSnapshot()
  })
})

const describeButtonStickyBottomComponent = '<ButtonStickyBottom /> test case'
describe(describeButtonStickyBottomComponent, () => {
  const { container } = render(
    <ButtonStickyBottom onClick={() => {}}>
      ButtonStickyBottom test
    </ButtonStickyBottom>
  )

  test(`Render ${describeButtonStickyBottomComponent} correctly`, () => {
    expect(container).toMatchSnapshot()
  })
})