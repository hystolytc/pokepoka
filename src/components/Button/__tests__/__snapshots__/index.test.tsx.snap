// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> test case Render <Button /> test case correctly 1`] = `
<div>
  <button
    class="css-g2mrbi"
  >
    Button test
  </button>
</div>
`;

exports[`<ButtonStickyBottom /> test case Render <ButtonStickyBottom /> test case correctly 1`] = `<div />`;
